server/src/controllers/notification_controller.ts

```diff
@@ -0,0 +1,41 @@
+import "reflect-metadata";
+import { Request, Response } from "restify";
+import { Controller, interfaces, Post } from "inversify-restify-utils";
+import { inject, injectable, named } from "inversify";
+import { NotificationService } from "../services/notification_service";
+import * as Joi from "joi";
+
+@Controller("/notifications")
+@injectable()
+export class NotificationController implements interfaces.Controller {
+    constructor(
+        @inject("Service")
+        @named("NotificationService")
+        private readonly notificationService: NotificationService,
+    ) {}
+
+    @Post("/sms", "injectAuthDataMiddleware")
```

Comment:
shouldn't this post be restricted to only a few roles?

Reply:
Yea probably I just don't know which roles should have access right now, I think its only health official because all the other ones will call the service directly through another service. So we can add the restrictions then.